src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadocmethod/InputMissingJavadocMethodAllowedAnnotations.java

```diff
@@ -28,9 +28,6 @@ public void allowed1() {}
     @ThisIsOkToo
     public void allowed2() {}
 
-    @com.puppycrawl.tools.checkstyle.checks.javadoc.missingjavadocmethod.ThisIsOk
-    public void allowed3() {}
-
```

Comment:
Please always provide explanation if you remove a test case.

Reply:
Better to never remove test input data.
We need all weird code, some duplicate is ok

Reply:
Input name InputMissingJavadocMethodAllowedAnnotations doesn't imply work without violation, please put violation over such code.

Reply:
> Please always provide explanation if you remove a test case.

I asked in the issue whether excluding `ThisIsOk` should also match `com.puppycrawl.tools.checkstyle.checks.javadoc.missingjavadocmethod.ThisIsOk` and the response I got was 
> To match as is

so the above case is a violation now. I will put it back and expect it in the test.

Reply:
Yes, just extend Input with violation comment.
There should be violation.